src/common.ts

```diff
@@ -53,25 +54,41 @@ export class WrappedContract {
   }
 }
 
+export enum ContractAddressId {
+  oracle = "ORACLE",
+  lens = "LENS",
+  adapter_ironbank = "REGISTRY_ADAPTER_IRON_BANK",
+  adapter_registry_v2 = "REGISTRY_ADAPTER_V2_VAULTS",
+  helper = "HELPER",
+  allowlist = "ALLOWLIST",
```

Comment:
```suggestion
  allowlist = "ALLOW_LIST",
```
just to be consistent